05LessonFive/Checkpoint1/index.html

```diff
@@ -1,10 +1,42 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title>Landscape Design</title>
-  <link rel="stylesheet" href="css/style.css">
+    <title>Berries</title>
+    <link rel="stylesheet" href="css/style.css">
 </head>
+
 <body>
 
+    <!--The div that encompasses both boxes-->
+    <div id="bothboxes">
+
+        <img src="img/berries.jpg" />
+        <!--berry image-->
+
+        <!--gray box on the right-->
+        <div id="graybox">
+
+            <!--All insides of box start here-->
+            <div id="boxinsides">
+                <div id="berrywords">
```

Comment:
instead of multiple ids, consider selecting things through their parent elements